pkg/controller/admissionchecks/provisioning/controller.go

```diff
@@ -287,6 +287,7 @@ func (c *Controller) syncOwnedProvisionRequest(ctx context.Context, wl *kueue.Wo
 			if err := c.client.Create(ctx, req); err != nil {
 				return nil, err
 			}
+			c.record.Eventf(wl, corev1.EventTypeNormal, "ProvisioningRequestCreated", "A new provisioning request %q was created", req.Name)
```

Comment:
```suggestion
			c.record.Eventf(wl, corev1.EventTypeNormal, "ProvisioningRequestCreated", "Created ProvisioningRequest: %v", req.Name)
```
nit: the word "new" is redundant. just to make the events more like others: https://github.com/kubernetes/kubernetes/blob/0c645922edcc06adff43c70c02fb56751364bbb5/pkg/controller/controller_utils.go#L598, or https://github.com/kubernetes-sigs/kueue/blob/74377c7eb3e37db4703538abbfb9432718be4097/pkg/controller/jobframework/reconciler.go#L801-L802.

Reply:
why %v?

Reply:
Just for the sake of consistency, feel free to use %q.

Reply:
done